test/command/suite/sharding/logical_select/columns/stage/filtered/window_function/window_count/ascending.test

```diff
@@ -0,0 +1,45 @@
+plugin_register sharding
+
+table_create Logs_20170415 TABLE_NO_KEY
+column_create Logs_20170415 timestamp COLUMN_SCALAR Time
+column_create Logs_20170415 price COLUMN_SCALAR UInt32
+column_create Logs_20170415 n_likes COLUMN_SCALAR UInt32
+
+table_create Logs_20170416 TABLE_NO_KEY
+column_create Logs_20170416 timestamp COLUMN_SCALAR Time
+column_create Logs_20170416 price COLUMN_SCALAR UInt32
+column_create Logs_20170416 n_likes COLUMN_SCALAR UInt32
+
+load --table Logs_20170415
+[
+{"timestamp": "2017/04/15 00:00:00", "n_likes": 2, "price": 100},
+{"timestamp": "2017/04/15 01:00:00", "n_likes": 1, "price": 100},
+{"timestamp": "2017/04/15 01:00:00", "n_likes": 2, "price": 200}
+]
+
+load --table Logs_20170416
+[
+{"timestamp": "2017/04/16 10:00:00", "n_likes": 1, "price": 200},
+{"timestamp": "2017/04/16 11:00:00", "n_likes": 2, "price": 300},
+{"timestamp": "2017/04/16 11:00:00", "n_likes": 1, "price": 300}
+]
+
+logical_select Logs \
+  --shard_key timestamp \
+  --filter 'price > 100' \
+  --columns[count].stage filtered \
+  --columns[count].type UInt32 \
+  --columns[count].flags COLUMN_SCALAR \
+  --columns[count].value 'window_count()' \
+  --columns[count].window.sort_keys _id \
+  --output_columns _id,n_likes,count
+
+logical_select Logs \
+  --shard_key timestamp \
+  --filter 'price > 100' \
+  --columns[count].stage filtered \
+  --columns[count].type UInt32 \
+  --columns[count].flags COLUMN_SCALAR \
+  --columns[count].value 'window_count()' \
+  --columns[count].window.sort_keys n_likes \
+  --output_columns _id,n_likes,count
```

Comment:
Why do we need two `logical_select`s in one test?

Reply:
This test for case of different order "sort keys" and "shard keys".

I thought this test separate is better.
Therefore, I'll make it as the other test.

Reply:
Should we output `price` instead of `price`?
It seems that `n_likes` isn't related to this `logical_select`.
(There are more same cases in these tests.)

Reply:
Ah...Sorry, I made a mistake.
I've modified that we output `price` instead of `n_likes`.